recipes/livvkit/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "livvkit" %}
+{% set version = "2.1.6" %}
+{% set nbuild = 0 %}
+{% set sha256 = "f6eba931d8cf9942512dcaa91f2d404c121faaafc3e1377f2c45ad19655fea88" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url:  https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }} 
+
+build:
+  noarch: python
+  number: {{ nbuild }}
```

Comment:
This probably shouldn't be a template parameter since it is only used once, and may interfere with the autoticker updating the feedstock.